Yurii_Telishevskyi/todo/src/app/todocomponent/todocomponent.component.html

```diff
@@ -0,0 +1,6 @@
+<div class="design1">
+    <input type="checkbox"  (change)="checked()">
+    <span [class.line]="isGood">{{ items }}</span>
+    <button [disabled]="isGood" (click)="isEdited()" class = "m-1 p-1 btn btn-outline-primary"> Edit </button>
```

Comment:
Name for click for example need contains action, for bool property start with "is"
(click) = "edit()"
[disable]="isEditable"